plugins/app-backend/src/lib/assets/StaticAssetsStore.ts

```diff
@@ -138,13 +138,19 @@ export class StaticAssetsStore implements StaticAssetProvider {
    */
   async trimAssets(options: { maxAgeSeconds: number }) {
     const { maxAgeSeconds } = options;
+    let lastModifiedInterval = `now() + interval '${-maxAgeSeconds} seconds'`;
+    if (this.#db.client.config.client.includes('mysql')) {
+      lastModifiedInterval = `date_sub(now(), interval ${maxAgeSeconds} second)`;
+    } else if (this.#db.client.config.client.includes('sqlite3')) {
+      lastModifiedInterval = `datetime('now', ?)`;
+    }
     await this.#db<StaticAssetRow>('static_assets_cache')
       .where(
         'last_modified_at',
         '<=',
         this.#db.client.config.client.includes('sqlite3')
```

Comment:
Instead of making this check twice, could you make the `lastModifiedInterval` variable to be of the type that `.raw` returns, instead of a string? Then you call `.raw` in three different ways depending on the engine

Reply:
Great suggestion, I was wondering what I should do to clean this up